components/Home.js

```diff
@@ -37,7 +37,7 @@ function Home() {
     <div className="font-pop dark:bg-gray-800 dark:text-white">
       <div>
         <div
-          className="px-10 md:px-72 text-center text-4xl md:text-5xl font-extrabold h-screen flex items-center text-gray-800"
```

Comment:
@tanya17-05  Add `px-10` class to the element to provide minimum padding otherwise, it may break in smaller screens.